.github/workflows/claude-preflight.yml

```
@@ -0,0 +1,321 @@
+      mode: ${{ steps.pr.outputs.mode || steps.comment.outputs.mode || steps.cron.outputs.mode }}
+      head_sha: ${{ steps.pr.outputs.head_sha || steps.comment.outputs.head_sha || steps.cron.outputs.head_sha }}
+      head_ref: ${{ steps.pr.outputs.head_ref || steps.comment.outputs.head_ref || steps.cron.outputs.head_ref }}
+
+    steps:
+      - name: Route pull_request
+        id: pr
+        if: github.event_name == 'pull_request'
+        env:
+          ACTOR: ${{ github.triggering_actor }}
+          PR_LABELS: ${{ toJSON(github.event.pull_request.labels.*.name) }}
+        run: |
```

actionlint (1.7.12)

[error] "github.event.pull_request.head.ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)